README.md

```diff
@@ -52,11 +52,11 @@ A strong complex password is at least 16 characters long (the longer the better)
 ## 2FA Audit
 Make sure 2FA or two step verification is enabled on all of the following accounts:
 
-- [ ] Google
+- [ ] Google - [Quick Jump](https://myaccount.google.com/security/signinoptions/two-step-verification)
```

Comment:
Hmm could we change to `[Google](https://myaccount.google.com/security/signinoptions/two-step-verification)` instead? I'm not a fan of the "quick jump" phrase. It reminds me of "click here" links - [here's a great overview of why you shouldn't use click here for links](http://stephanieleary.com/2015/05/why-click-here-is-a-terrible-link-and-what-to-write-instead/).